Resources/doc/entity-listeners.rst

```diff
@@ -14,11 +33,45 @@ entity manager it should be registered with. Example:
             user_listener:
                 class: \UserListener
                 tags:
+                    # Minimal configuration
                     - { name: doctrine.orm.entity_listener }
+                    # Or, optionally, you can give the entity manager name
+                    - { name: doctrine.orm.entity_listener, entity_manager: custom }
+    .. code-block:: xml
+
+        <?xml version="1.0" ?>
+
+        <container xmlns="http://symfony.com/schema/dic/services"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+
+            <services>
+                <service id="user_listener" class="UserListener">
+                    <tag name="doctrine.orm.entity_listener" event="preUpdate" entity="App\Entity\User" />
```

Comment:
Since we're using the minimal configuration in YAML, the same configuration should be given here as well.

Reply:
Fixed